Sources/Swift/SwiftDescriptor.swift

```diff
@@ -7,7 +8,15 @@ class SwiftDescriptor: NSObject {
     static func getObjectClassName(_ object: AnyObject) -> String { 
         return String(describing: type(of: object))
     }
-    
+
+    @objc
+    static func getViewControllerClassName(_ object: UIViewController) -> String {
+        if let object = object as? SentryUIViewControllerDescriptor {
+            return object.sentryName
+        }
+        return String(describing: type(of: object))
```

Comment:
`m`: Less duplication:
```suggestion
        return getObjectClassName(object)
```